DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Receive/RhinoHostObjectBuilder.cs

```diff
@@ -38,25 +38,18 @@ CancellationToken cancellationToken
     var baseLayerName = $"Project {projectName}: Model {modelName}";
 
     var objectsToConvert = _traverseFunction
-      .Traverse(rootObject)
-      .Where(obj => obj.Current is not Collection)
-      .Select(ctx => (GetLayerPath(ctx), ctx.Current))
-      .ToArray();
+      .TraverseWithProgress(rootObject, onOperationProgressed, cancellationToken)
```

Comment:
Do we report progress on Traversing to UI?

Reply:
The actually traversal step, does not get reported with progress because its super quick and only happens once.

 `TraverseWithProgress`, since it returns a lazily enumerated `IEnumerable` reports the actual conversion progress (as the enumeration of objects occurs)

This was simply my hope to be DRY with the progress reporting, but I'm not tied to it if we find a better way.